pom.xml

```
@@ -6,7 +6,7 @@
     <groupId>com.idrsolutions</groupId>
     <artifactId>buildvu-microservice-example</artifactId>
     <packaging>war</packaging>
-    <version>2.2.0-alpha</version>
+    <version>3.2.0-alpha</version>
```

Could this be changed back to 2.3.0 before merging?

```suggestion
    <version>2.3.0-alpha</version>
```